index.html

```diff
@@ -762,9 +762,311 @@
         <!-- ************ -->
 
         <section id="class-3">
-          <h2>Third Class: Jeff</h2>
+          <h2>Third Class</h2>
+          <p>Jeff O'Leary</p>
         </section>
+        <section>
+          <h4>Agenda</h4>
+          <p class="fragment">Navigation</p>
```

Comment:
How much of a review were you thinking?  I was planning on making connections between what I'm teaching now and reviewing what has already been gone over, to connect the dots.  I'd rather get through the 2 topics tomorrow night than do a full review beforehand and not get to autolayout.